.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -6,11 +6,10 @@ body:
     attributes:
       label: 📚 The doc issue
       description: >
-        A clear and concise description of what content in within the inline documentation has an issue. We should have
-        additional documentation on the PyTorch website soon. If this has to do with the general https://pytorch.org
-        website, please file an issue at https://github.com/pytorch/pytorch.github.io/issues/new/choose instead. If this
-        has to do with https://pytorch.org/tutorials, please file an issue at
-        https://github.com/pytorch/tutorials/issues/new.
+        A clear and concise description of what content in https://pytorch.org/data is an issue. If this has to do with
+        the general https://pytorch.org website, please file an issue at
+        https://github.com/pytorch/pytorch.github.io/issues/new/choose instead. If this has to do with
+        https://pytorch.org/tutorials, please file an issue at https://github.com/pytorch/tutorials/issues/new.
```

Comment:
As we are providing tutorial on https://pytorch.org/data/main/tutorial.html, we should still let users to file issue in TorchData.